tests/toplevel_unit.c

```diff
@@ -120,6 +120,41 @@ void test_example() {
     roaring_bitmap_free(r3);
 }
 
+bool check_bitmap_from_range(uint32_t min, uint32_t max, uint32_t step) {
+    roaring_bitmap_t *result = roaring_bitmap_from_range(min, max, step);
+    assert_non_null(result);
+    roaring_bitmap_t *expected = roaring_bitmap_create();
+    assert_non_null(expected);
+    for(uint32_t value = min ; value < max ; value += step) {
+        roaring_bitmap_add(expected, value);
+    }
+    bool is_equal = roaring_bitmap_equals(expected, result);
+    if(!is_equal) {
+        fprintf(stderr, "[ERROR] check_bitmap_from_range(%u, %u, %u)\n",
+            (unsigned)min, (unsigned)max, (unsigned)step);
+    }
+    return is_equal;
+}
+
+void test_bitmap_from_range() {
+    assert_true(roaring_bitmap_from_range(1, 10, 0) == NULL); // undefined range
+    bool no_error = true;
+    no_error = check_bitmap_from_range(5, 1, 3) && no_error; // empty range
```

Comment:
The idea of this code was to run all the tests related to `roaring_bitmap_from_range` and output a message for every eventual error. The goal is to ease the debugging.

If I modify the code like you say, then : 
* The test stops after one error is detected.
* We don't know which sub-test failed (i.e. the values of `min`, `max` and `step`).

Do you have any idea on how to avoid these issues? I think the second is the most important.